bin/runtime/src/lib.rs

```diff
@@ -1015,10 +1015,35 @@ impl_runtime_apis! {
 
 #[cfg(test)]
 mod tests {
-    use crate::VERSION;
+    use super::*;
+    use primitives::HEAP_PAGES;
+    use smallvec::Array;
```

Comment:
Where do we use this exactly? IIRC, `smallvec` gives _some_ performance gains by pre-allocating space up to some small numbers of vec elements. I'm not sure it's worth adding new dependency just for it, especially in tests.

Reply:
We can maybe move to `dev-dependencies`?

Reply:
@deuszx `CallStack` is of this type so I need to import it to call `CallStack::size`.
@DamianStraszak sure.